tests/test-team/README.md

```diff
@@ -0,0 +1 @@
+# NHS Notify Template Management (WebUI) Automated Regression Tests
```

Comment:
this empty README doesn't seem very useful - either add to it or remove it

Reply:
Might be a good place to talk about the difference between the e2e tests and component tests and where the boundary is between them?

Reply:
added